Makefile

```diff
@@ -763,13 +763,17 @@ edpm_deploy_baremetal: input edpm_deploy_baremetal_prep ## installs the dataplan
 .PHONY: edpm_wait_deploy_baremetal
 edpm_wait_deploy_baremetal: edpm_deploy_baremetal ## waits for dataplane readiness. Runs prep step in advance. Set DATAPLANE_REPO and DATAPLANE_BRANCH to deploy from a custom repo.
 	$(eval $(call vars,$@,dataplane))
-	oc kustomize ${DEPLOY_DIR} | oc wait --for condition=Ready --timeout=$(BAREMETAL_TIMEOUT) -f -
+	oc kustomize ${DEPLOY_DIR}
+	oc wait --for condition=Ready OpenStackDataPlaneDeployment --all --timeout=$(BAREMETAL_TIMEOUT)
+	oc wait --for condition=Ready OpenStackDataPlaneNodeSet --all --timeout=$(BAREMETAL_TIMEOUT)
```

Comment:
Do we need the `oc kustomize ${DEPLOY_DIR}`?  I'm trying to understand what purpose it serves without passing its result to the old `oc wait`.

Also, I guess since these EDPM deploy targets ultimately depend on (and execute) `edpm_deploy_cleanup`, the cluster normally won't have any extra `OpenStackDataPlaneDeployments` or `OpenStackDataPlaneNodeSets` lingering.  If the user manually created additional ones, however, things would get stuck here with the `--all` flag.  But I think it's fine for now.

Reply:
> Also, I guess since these EDPM deploy targets ultimately depend on (and execute) edpm_deploy_cleanup, the cluster normally won't have any extra OpenStackDataPlaneDeployments or OpenStackDataPlaneNodeSets lingering. If the user manually created additional ones, however, things would get stuck here with the --all flag. But I think it's fine for now.

Yeah, I either wait for every CR of these types or I need a way to know the names of the CRs to wait for. The latter seemed more complicated / fragile as any sample can be passed on install_yamls. Let me see if I can come up with something reasonable. as I anyhow need to respin this.

Reply:
What about a nice yq filter on the kustomize output?  :)